src/QuickMarcEditor/QuickMarcEditor.js

```diff
@@ -295,6 +312,20 @@ const QuickMarcEditor = ({
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [httpError]);
 
+  useEffect(() => {
+    if (confirmRemoveAuthorityLinking) {
+      const authorityRecordIndex = records.findIndex(record => (record.tag === '100' && record._isLinked));
```

Comment:
The modal needs to unlink all fields, not just `100`

Reply:
Isn't not just 100 field associated with authority record?